spaces/bzx/skin.scss

```
@@ -0,0 +1,10 @@
+.name {
```

The class name should be the same as the space key
```suggestion
.bzx {
```
And the file should be stored in the repo root "/skins/bzx.scss". 
Also you should add a field "skin": "bzx" in the index.json file.